EasyCommands/Program.cs

```diff
@@ -160,8 +160,9 @@ public partial class Program : MyGridProgram {
         void RunThreads() {
             try {
                 // invalidate block & group cache
-                invalidBlockCache = true;
-                invalidGroupCache = true;
+                blockCache.Clear();
+                groupCache.Clear();
+                selectorCache.Clear();
```

Comment:
Clever that we aren't clearing the actions/properties.  This would definitely improve script performance when using actions/properties directly.

Reply:
we might want think about keeping (some of) these caches alive for multiple ticks. Every 10th tick would still refresh the caches six times per second.

Reply:
I think we hold off on that one for now.  Users can change the run frequency and there's no guarantee that the program is running constantly (buttons to do commands), so the caches would have to be refreshed based on a "last updated time".

I think caching per tick is fine for now.